docs/blog/author.yaml

```diff
@@ -304,3 +304,7 @@
   bio: Writes things @ GatsbyJS. Chaotic good frontend dev, caffeine addict, backyard chicken wrangler 🐓.
   avatar: avatars/michelle-gienow.jpg
   twitter: "@michellegienow"
+- id: Anthony Marcar
+  bio: "todo: write this"
```

Comment:
```suggestion
  bio: "Software consultant, Clojure enthusiast, and founder of Grabble (acquired by Walmart Labs)."
```

Reply:
^ (if helpful ;)) 

listened to your podcast to see how you'd introduce yourself there and then you dodged the question 😂

Reply:
Only just saw this, you were very close to what I ended up writing (just replace bicycle tourer with founder ;) )